From dfe6a838f9824546c6e4ce80f436980034c91aca Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Tue, 26 Jul 2016 20:25:09 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-evb: add utmi-avalid for
 evb1/evb2

The usb2 phy utmi bvalid status is invalid for ES1 evb1/evb2
board, so check vbus status by utmi avalid status.

Change-Id: Iae369183cd15034eb35818c63988150f3aad6374
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi | 4 ++++
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
index 7fe93200650d..41b1da42fff1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1.dtsi
@@ -296,3 +296,7 @@
 &pwm2 {
 	status = "okay";
 };
+
+&u2phy0_otg {
+	rockchip,utmi-avalid;
+};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index 89043adc5dbe..f92dc485b943 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -447,3 +447,7 @@
 		};
 	};
 };
+
+&u2phy0_otg {
+	rockchip,utmi-avalid;
+};
-- 
2.35.3

